### PR TITLE
Include open tickets as @contact.tickets

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: core/models/tickets.go [hunk 1]
```diff
@@ -101,7 +101,9 @@ func (t *Ticket) ForwardIncoming(ctx context.Context, db Queryer, org *OrgAssets
 	logger := &HTTPLogger{}
 	err = service.Forward(t, msgUUID, text, attachments, logger.Ticketer(ticketer))
 
-	return logger.Insert(ctx, db)
+	logger.Insert(ctx, db)
+
+	return err
```
Comment (rowanseymour) [inline core/models/tickets.go:106]: unrelated fix - just saw warning that `err` was not used

File: core/models/contacts.go [hunk 2]
```diff
@@ -498,6 +520,25 @@ LEFT JOIN (
...
+	SELECT
+		contact_id,
+		array_agg(
+			json_build_object(
+				'uuid', t.uuid,
+				'subject', t.subject,
+				'body', t.body,
+				'external_id', t.external_id,
+				'ticketer_id', t.ticketer_id
+			) ORDER BY t.opened_on ASC, t.id ASC
```
Comment (rowanseymour) [inline core/models/contacts.go:533]: @nicpottier note here that tickets are ordered chronologically. This should be hitting the `tickets_contact_open` index tho need to verify that.

Comment (rowanseymour) [reply core/models/contacts.go:533]: Have checked and it is using `tickets_contact_open`

**Review by nicpottier — APPROVED** [review]
woot!
